packages/dx-react-grid/docs/guides/grouping.md

```diff
@@ -58,6 +58,12 @@ In [controlled mode](controlled-and-uncontrolled-modes.md), pass a grouping opti
 
 .embedded-demo(grid-grouping/controlled-mode)
 
+## Disable Grouping by a Column
+
+The [GroupingState](../reference/grouping-state.md) plugin's `columnExtensions` property allows you to prevent grouping by a specific column.
```

Comment:
This description seems a bit weird. It states `columnExtensions` is a property designated for controlling grouping availability. But it is a more generic config which can handle more tasks in the future. Let's try to rephrase this somehow.

Reply:
Yes, `columnExtensions` may allow(in the future) configuring not only grouping availability. But this section is only about column grouping availability("Disable Grouping by a Column"). It's the most notable case. And, for now, we don't provide any other column extensions for this plugin.

Reply:
Still, it should not sound like there is only a single purpose for it even if it is true for now :)

Reply:
Well, we already use the similar description for filtering & column visibility. @viterobk is it OK?

Reply:
One more point) This section doesn't describe the `columnExtensions` interface. It describes how to use only part of it. The `columnExtensions` is completely described in api reference.

Reply:
The current wording can make a reader think that preventing grouping by a column is the only purpose of the `GroupingState` plugin's `columnExtensions` property. If we are going to extend its functionality,  it's better to say like "Prevent grouping by a specific column using the `GroupingState` plugin's `columnExtensions` property".